includes/writer.js

```diff
@@ -146,11 +148,11 @@ module.exports = function(couchDbUrl, bufferSize, parallelism) {
             done();
           });
         } else {
-          console.error('ERROR on line', linenumber, ': not an array');
+          ee.emit('error', new error.BackupError('CorruptBackup', `Error on line ${linenumber} of backup file - not an array`));
           done();
         }
       } catch (e) {
-        console.error('ERROR on line', linenumber, ': cannot parse as JSON');
+        ee.emit('error', new error.BackupError('CorruptBackup', `Error on line ${linenumber} of backup file - cannot parse as JSON`));
```

Comment:
I wish I could think of a better word than `Corrupt` to use here. Whilst there is a corrupted batch in the file if this error is thrown, as you know it is non-fatal and could have no impact on the completeness of what is being restored if it was a resumed backup.